pkg/cleaner/jobcontroller.go

```diff
@@ -83,8 +83,12 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, tolerations []v
 		// wipefs erases the filesystem signature from the block
 		// -a    wipe all magic strings
 		// -f    force erasure
-		args := fmt.Sprintf("(fdisk -o Device -l %s | grep \"^%s\" | xargs -I '{}' wipefs -fa '{}') && wipefs -fa %s && partprobe %s",
-			bd.Spec.Path, bd.Spec.Path, bd.Spec.Path, bd.Spec.Path)
+		args := fmt.Sprintf("(fdisk -o Device -l %[1]s "+
```

Comment:
Is there any way we can add unit tests to verify that the grep expressions actually is accurate. 
an idea could be to simulate different outputs passed to the grep command ... compare the result of the grep command.

Reply:
Comparing results means, we will have to execute the command right?

Reply:
We can write a normal unit test case for each of the different types of blockdevices and the volume mode. But in these tests only `container.args` will be compared with the expected output. 

I think this is not what @kmova want?

Reply:
How do we ensure/validate that the following lines are picking the right set of devices in different use cases?

```
fdisk -o Device -l %[1]s | grep \"^%[1]s\"
```

Reply:
looks like a bit of magic sauce in there. probably comments to explain what each subcommand does also might increase some confidence.

Reply:
Added the comments.